src/responders/esi.ts

```diff
@@ -27,6 +27,7 @@ export const handle = (m: Message): void => {
         })
         .catch(e => {
           log.error(`failed to call /status endpoint with error: ${e}`);
+          m.channel.send(`${m.author}, /status endpoint from ESI ended with error: ${e}.`);
```

Comment:
Can we remove the `{e}` here? I am afraid of sensitive information getting dumped in Discord.  Rather let's have it just say something like "refer to the logs for more information" or something.